tests/test_vert_coords.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import pytest
+
+from pyaerocom.griddeddata import GriddedData
+from pyaerocom.vert_coords import *
+from tests.conftest import lustre_unavail
+from tests.fixtures.tm5 import data_tm5
+
+
+@pytest.fixture
+def alt(data_tm5: GriddedData) -> AltitudeAccess:
+    return AltitudeAccess(data_tm5)
+
+
+@pytest.fixture
+def sin_wave() -> np.ndarray:
+    return np.sin(np.arange(1000))
+
+
+def test_atmosphere_sigma_coordinate_to_pressure_exceptions():
+    with pytest.raises(ValueError):
+        atmosphere_sigma_coordinate_to_pressure(1.0, 0, "")
+
+
+@pytest.mark.parametrize(
+    "sigma,ps,ptop",
+    (
+        pytest.param(1.0, 0, 0.0, id="ptop-is-float"),
+        pytest.param(1.0, 0, np.ones(10), id="ptop-is-ndarray"),
+    ),
+)
+def test_atmosphere_sigma_coordinate_to_pressure(sigma, ps, ptop):
+    result = atmosphere_sigma_coordinate_to_pressure(sigma, ps, ptop)
+
+    assert type(result) == type(ptop)
+    if isinstance(result, np.ndarray):
+        assert len(result) == len(ptop)
```

Comment:
Is it possible to check the result on the `else`?
Also, what should be the value?
You can numerical values with `assert result == pytest.approx(expected_reesult)`

Reply:
I've added expected values, and an additional testcase using a sine-wave to cover a broader range of the function domain.